pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -41,10 +41,10 @@ type AgentCommand string
 // Aliases are also used as keys to the store, it does not matter how the keys are named,
 // only whether it relates to the right data
 const (
-	GetFilesystem     AgentCommand = "guest-get-fsinfo"
 	GetAgent          AgentCommand = "guest-info"
 	GetFSFreezeStatus AgentCommand = "guest-fsfreeze-status"
 
+	FSInfoKey           = "FSInfo"
```

Comment:
I'm not sure about this constant, can you just use `libvirt.DOMAIN_GUEST_INFO_FILESYSTEM` instead? I think it would be better to keep it consistent with others keys from libvirt package.

@akrejcir WDYT?